### DISCUSSION TITLE
can i use multiple theme

### DISCUSSION THREAD

**Discussion by mdrakash** [primefaces/community#723] [PrimeVue] [answered]
i want to use multiple theme in my component . like my a component use light theme and b component use dark theme . its possible ? how can I do this using primevue

**Comment by tugcekucukoglu** [comment, marked as answer]
Yes, it's possible. `pt` property and unstyled mode can provide this. [Overriding Presets section](https://tailwind.primevue.org/guides/overriding-presets/) might be a good starter.
